Library/Formula/aeskeyfind.rb

```diff
@@ -0,0 +1,60 @@
+class Aeskeyfind < Formula
+  homepage "https://citp.princeton.edu/research/memory/code/"
```

Comment:
It's a stable purpose-built tool that works well and is included in popular distros:

https://packages.debian.org/search?keywords=aeskeyfind
http://packages.ubuntu.com/search?keywords=aeskeyfind